zcnbridge/transaction/txn.go

```diff
@@ -22,6 +24,30 @@ type (
 	}
 )
 
+type (
+	verifyOutput struct {
+		Confirmation confirmation `json:"confirmation"`
+	}
+
+	// confirmation represents the acceptance that a transaction is included into the blockchain.
+	confirmation struct {
```

Comment:
why can't we use core transaction completely?

Reply:
It's different